OneSignalExample/Assets/OneSignal/src/OneSignal.cs

```diff
@@ -1250,7 +1250,7 @@ public class UnityBuilder {
 
         // Some functions have a 'success' and 'failure' delegates, so this validates nothing is missing the json response
         private bool isValidSuccessFailureDelegate(Dictionary<string, object> jsonObject) {
-            if (!isValidDelegate(jsonObject))
+            if (jsonObject==null || !isValidDelegate(jsonObject) || !jsonObject.ContainsKey("delegate_id"))
```

Comment:
`!jsonObject.ContainsKey("delegate_id")` is not needed, `isValidDelegate` already checks for this.

Reply:
`jsonObject==null` should have a spaces around the `==`
`jsonObject == null`